packages/core/src/payment/strategies/worldpayaccess/worldpayaccess-payment-strategy.ts

```diff
@@ -9,97 +9,122 @@ import { WorldpayAccess3DSOptions, WorldpayAccessAdditionalAction, WorldpayAcces
 
 const IFRAME_NAME = 'worldpay_hosted_payment_page';
 const IFRAME_HIDDEN_NAME = 'worldpay_hosted_hidden_payment_page';
-const INVALID_URL = 'invalid url';
+const PAYMENT_CANNOT_CONTINUE = 'Payment cannot continue';
+
+let submit: Function;
```

Comment:
Also instead of making this as a global variable, why not make it a class private member?

Reply:
because as private member we need to be initialized in the constructor or allow to be optional adding complexity to the code.